CHANGELOG.next.asciidoc

```diff
@@ -227,6 +227,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix panic on `metricbeat test modules` when modules are configured in `metricbeat.modules`. {issue}18789[18789] {pull}18797[18797]
 - Fix getting gcp compute instance metadata with partial zone/region in config. {pull}18757[18757]
 - Add missing network.sent_packets_count metric into compute metricset in googlecloud module. {pull}18802[18802]
+- Fix ECS compliance of user.id field in system/users  metricset {pull}19019[19019]
```

Comment:
Changing the type of a field looks like a breaking change.

Reply:
Bah, you're probably right.